DOCUMENTATION.md

```diff
@@ -722,23 +693,74 @@ If the request to the client is successful, the client returns a `dict`.
 <generator object NotificationsAPIClient.get_all_notifications_iterator at 0x1026c7410>
 ```
 
+For more information, see the:
+
+* [email status descriptions](#status-email)
+* [text message status descriptions](#status-text-message)
+* [letter status descriptions](#status-letter)
+* [precompiled letter status descriptions](#status-precompiled-letter)
+
 ### Error codes
 
 If the request is not successful, the client returns an `HTTPError` containing the relevant error code:
 
 |error.status_code|error.message|How to fix|
 |:---|:---|:---|
-|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "bad status is not one of [created, sending, delivered, pending, failed, technical-failure, temporary-failure, permanent-failure]"`<br>`}]`|Contact the GOV.UK Notify team|
-|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "Apple is not one of [sms, email, letter]"`<br>`}]`|Contact the GOV.UK Notify team|
-|`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Error: Your system clock must be accurate to within 30 seconds"`<br>`}]`|Check your system clock|
-|`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Invalid token: API key not found"`<br>`}]`|Use the correct API key. Refer to [API keys](#api-keys) for more information|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "‘bad status’ is not one of [created, sending, delivered, pending, failed, technical-failure, temporary-failure, permanent-failure]"`<br>`}]`|Change the [status argument](#status-optional)|
```

Comment:
@servingUpAces does this error message accurately reflect the list of statuses given in the API?